autoload/OmniSharp/locations.vim

```diff
@@ -85,24 +85,41 @@ function! OmniSharp#locations#Navigate(location, ...) abort
   return navigated
 endfunction
 
+function! OmniSharp#locations#ParseLocation(value) abort
+  let filename = substitute(a:value.FileName, '?', '', 'g')
```

Comment:
What is this substitution for? I thought maybe metadata but this codepath is never hit when parsing a metadata response.

Reply:
I removed this.